bin/nixb

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+nix eval --json .#targets.$system | jq -r ".[]" | fzf --bind 'enter:become(echo "Running nix build .#{}" && nix build .#{})'
```

Comment:
Super nitpicking and purely a matter of taste so ignore if you wish: I like if the `--json` is close to the `jq` parsing like:
```suggestion
nix eval .#targets.$system --json | jq -r ".[]" | fzf --bind 'enter:become(echo "Running nix build .#{}" && nix build .#{})'
```